Pcap++/header/PfRingDeviceList.h

```diff
@@ -46,7 +48,9 @@ namespace pcpp
 		/**
 		 * Return a list of all available PF_RING devices
 		 * @return a list of all available PF_RING devices
+		 * @deprecated This method has been deprecated in favor of direct accessor API.
 		 */
+		PCPP_DEPRECATED("Deprecated in favor of direct accessor API")
 		const std::vector<PfRingDevice*>& getPfRingDevicesList() const { return m_PfRingDeviceListView; }
```

Comment:
You missed updating it in the tests: https://github.com/Dimi1010/PcapPlusPlus/blob/2ae51d24cc81103afa70d28995011cfe965d829e/Tests/Pcap%2B%2BTest/Tests/PfRingTests.cpp#L203

Reply:
Updated ab45201